### DISCUSSION TITLE
Solid green light; reuploading sketch fails

### DISCUSSION THREAD

**Discussion by FlitPix** [sanni/cartreader#387] [Need help]
Hi! I learned how to solder just for this project and I'm overall happy with the result; however, I'm having trouble with the HW5 Rev3 version of the cartreader.

When I connect it to power, the Arduino simply has a solid green light and nothing else happens. The display unit doesn't power on, either. I thought it might be an issue with the sketch, so I plugged it into my PC to try and reupload it, however it errors:
```
avrdude: stk500v2_ReceiveMessage(): timeout
avrdude: stk500v2_ReceiveMessage(): timeout
avrdude: stk500v2_ReceiveMessage(): timeout
avrdude: stk500v2_ReceiveMessage(): timeout
avrdude: stk500v2_ReceiveMessage(): timeout
avrdude: stk500v2_ReceiveMessage(): timeout
avrdude: stk500v2_getsync(): timeout communicating with programmer
An error occurred while uploading the sketch
```
Of course, setting the switch to 3.3V does nothing, which matches other reports. I don't think that's related to my issue, though.

A few things to note:
- I do not have an N64 cartridge slot or controller port installed in the six slot adapter. There were none in stock for the former, and I forgot to install the latter.
- I uploaded the sketch (twice, to make sure) before I desoldered the fuse from the Arduino and proceeded with building the reader.

Any ideas as to what I might try at this point?

**Comment by sanni** [comment]
You could try flashing it without the LCD, the cart slot adapter and the clock generator plugged in to see if those are maybe the issue. The Cart Reader's switches need to be set to 5V and ON. 

I'm still waiting for my Rev3 PCB, once it arrives I will get back to you.

**Comment by FlitPix** [comment]
Looking at the "finished" product I realize that I've made several glaring errors with this first attempt. I've ordered a new set of components to try again. That's probably why I'm running into issues.

I'll try and get this second attempt hopefully running smoothly!
